ios/sound.mm

```diff
@@ -46,13 +46,27 @@
             {
                 // ok
             }
+            else
+            {
+                // TODO - alert user
+            }
         }];
         [[AVAudioSession sharedInstance] setMode:AVAudioSessionModeMeasurement error:&audioSessionError];
     }
     catch ( const CGenErr& generr )
     {
-        qDebug ( "Sound exception ...." );
+        QMessageBox::warning( nullptr, "Sound exception", generr.GetErrorText() );
     }
+
+    buffer.mNumberChannels    = 2;
+    buffer.mData              = malloc ( 256 * sizeof ( Float32 ) * 2 ); // max size
```

Comment:
Is `2` in the `* 2` the same as the `= 2` on the previous line?  If it is, could you use `buffer.mNumberChannels` to be clearer?

Also, what's "m" on the front of the variable names?  I've not seen it in the code base before?

Reply:
Yes it’s the same. Thanks I’ll update the code.

“m” is from Apple: https://developer.apple.com/documentation/coreaudiotypes/audiobuffer/1421973-mnumberchannels?changes=l_3_8&preferredLanguage=occ

Reply:
Ah right.